### PR TITLE
notes

### PR DESCRIPTION
**Pull request by andrewreshitko**

### CODE REVIEW

File: content/_releases/23/31.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  - /releases/23.31.html
+---
+
+## New Features
+### Dashboard Redesign:
+* Added Ordinary Flows counter - *click on the counter to open `Flows` page with filtered ordinary flows*
+* Added Real-time Flows counter - *click on the counter to open `Flows` page with filtered real-time flows*
+* Redesigned the current Execution Statistics graph - *each day on the graph is clickable and leads users to the corresponding page*
+* Added new graph for Container errors - *each day on the graph is clickable and leads users to the corresponding page*
+* Transferred Workspace Memory Usage graph
```
Comment (KMobbs) [inline content/_releases/23/31.md:19]: Transferred the QUOTA usage graph from the QUOTA page. QUOTA page is now deleted.

File: content/_releases/23/31.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+### Transfer modal windows to the new styles:
+* Transform UI for Flow Designer: Configuration for Content-Based Router Component to React - Now users can map values from previous steps for evaluation.
```
Comment (KMobbs) [inline content/_releases/23/31.md:22]: Flow designer UI changes: Configuration screens for the Content Based Router are changed to Reat & now Users can map values from previous steps for evaluation

File: content/_releases/23/31.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+### Removed `LOOKOUT_PROCESS_DATA` env var
+`LOOKOUT_PROCESS_DATA` was introduced to disable Runlog for the dashboard. After redesigning the Dashboard page, this env var does not do anything on the frontend, since runlog is not on the new dashboard.
+
+We've removed this env var from the frontend completely, and removed it from a lookout by making it run as if this env var is set to "false".
+
+### Change elasticsearch index config:
+
+We've improved the Logs search to better handle partial text matching (e.g. "test.123" log line previously would match only if a user searched for exact "test.123", but now both "test" and "123" will match the logline).
+
+Also, we've fixed the bug when irrelevant search results appeared due to incorrect configuration of Elasticsearch.
```
Comment (KMobbs) [inline content/_releases/23/31.md:38]: when -> where

File: content/_releases/23/31.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+## OEM Related
+### Removed `LOOKOUT_PROCESS_DATA` env var
+`LOOKOUT_PROCESS_DATA` was introduced to disable Runlog for the dashboard. After redesigning the Dashboard page, this env var does not do anything on the frontend, since runlog is not on the new dashboard.
+
+We've removed this env var from the frontend completely, and removed it from a lookout by making it run as if this env var is set to "false".
+
+### Change elasticsearch index config:
+
+We've improved the Logs search to better handle partial text matching (e.g. "test.123" log line previously would match only if a user searched for exact "test.123", but now both "test" and "123" will match the logline).
```
Comment (KMobbs) [inline content/_releases/23/31.md:36]: The search function is now more user friendly as it now accepts partial matches (e.g.  ...

File: content/_releases/23/31.md (hunk 1 above)
Comment (HannaTrotsenko) [reply content/_releases/23/31.md:19]: Quota page is not deleted. We've transfered just one part - workspace usage with delatalized info per flow.

Comment (HannaTrotsenko) [reply content/_releases/23/31.md:19]: Also we can add that when user clicks the workspace name in Workspace Memory Usage table at Current Month Quota
 page he/she will be redirected to the  Dashboard of corrwsponding workspace

Comment (andrewreshitko) [reply content/_releases/23/31.md:19]: * Transferred the Workspace Quota Usage graph from the Quota page - *clicking the workspace name in the `Workspace Memory Usage` list on the `Current Month Quota` page will redirect the user to the corresponding workspace Dashboard.*

**Review by HannaTrotsenko — APPROVED** [review]

**Pull request merged**